src/include/ceph_features.h

```diff
@@ -84,6 +84,7 @@
 // duplicated since it was introduced at the same time as CEPH_FEATURE_CRUSH_TUNABLES5
 #define CEPH_FEATURE_NEW_OSDOPREPLY_ENCODING (1ULL<<58) /* New, v7 encoding */
 #define CEPH_FEATURE_FS_FILE_LAYOUT_V2       (1ULL<<58) /* file_layout_t */
+#define CEPH_FEATURE_FS_BTIME                (1ULL<<59) /* btime */
```

Comment:
I know @athanatos has done some work on feature bit reclamation; we should ask him or @liewegas prior to merging something that uses one of our last few.

Actually we might be better off to put btime support in the MDSMap CompatSet stuff, since it should really be a cluster thing. And the admin can decide if they want to let in clients which don't support it or not. Hmm...